src/screens/Documents/Modify/index.js

```diff
@@ -31,6 +25,10 @@ type Props = {
    * @desc React Native Navigation navigator object.
    */
   navigator: Navigator,
+  /**
+   * @desc Function to close menu modal.
+   */
+  onCancel: () => void,
```

Comment:
Naming seems to be not matching with actual sense of the function. Since it's called not only on cancel. Maybe something like `onWillClose` is better?

Reply:
Done